src/sql/src/plan/statement/ddl.rs

```diff
@@ -1038,22 +1041,27 @@ fn get_encoding_inner<T: sql_parser::ast::AstInfo>(
                 })
             }
         }
-        Format::Protobuf {
-            message_name,
-            schema,
-        } => {
-            let descriptors = match schema {
-                sql_parser::ast::Schema::Inline(bytes) => strconv::parse_bytes(&bytes)?,
-                sql_parser::ast::Schema::File(_) => {
-                    unreachable!("File schema should already have been inlined")
-                }
-            };
+        Format::Protobuf(schema) => match schema {
+            ProtobufSchema::Csr { .. } => {
+                unsupported!("confluent schema registry protobuf schemas");
```

Comment:
Putting the `unsupported!` here instead of in `plan_create_source` or `plan_create_sink` is a bit hacky! But it means that I can submit this PR without adding Confluent Schema Registry Protobuf schemas to our `testdrive` actions. (But that will be next!)

Reply:
Isn't this directly called by `plan_create_source`? Putting it here seems right to me!